README.md

```diff
@@ -92,6 +92,10 @@ const reporter = require('wdio-reportportal-reporter')
 * `reporter.addAttributeToCurrentSuite({key, value})` - add an attribute to current suite.
   * `key` (*string*, optional) -  attribute key. It must be non-empty string.
   * `value` (*string*, required)–  attribute value. It must be non-empty string.
+* `reporter.addDescriptionToCurrentSuite(description)` - add some string to current suite.
+  * `description` (*string*) - description content
```

Comment:
Please add info that description string could be formated via markdown markup language